abcgo.sh

```diff
@@ -26,7 +26,11 @@ if [[ $* == *verbose* ]]; then
     VERBOSE=true
 fi
 
-GO111MODULE=off go get -u github.com/droptheplot/abcgo
+if ! [ -x "$(command -v abcgo)" ]
```

Comment:
But then `-u` options doesn't make any sense

Reply:
yes that's true

Reply:
OT: I really miss sensible `-v` on `go get`